applications/modmesh/twnia1_env.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#PBS -P <PROJECTSYSTEMCODE>
+#PBS -N build_modmesh_env
+#PBS -l select=1:ncpus=8:mpiprocs=8
```

Comment:
Values in the resource list and the resource list itself should be parameters in the future.